<?xml version="1.0"?>
<launch>
  <arg name="estimation_rate" default="30"/>
  <arg name="odometry_topic" default="/vins_fusion/imu_propagate" />
  <arg name="servo_odom_topic" default=""/>
  <arg name="sensor_pose_topic" default="/pcl_render_node/camera_pose" />
  <arg name="servo_pose_topic" default=""/>

  <node pkg="test_temporarily" type="sensor_pose_trans_node" name="sensor_pose_trans_node">
    <param name="estimation_rate" value="30.0" />

    <remap from="~odom" to="$(arg odometry_topic)" />
    <remap from="~servo_odom" to="$(arg servo_odom_topic)"/>

    <remap from="~sensor_pose" to="$(arg sensor_pose_topic)" />
    <remap from="~servo_pose" to="$(arg servo_pose_topic)"/>
  </node>


</launch>